patterns/project-layout.php

```diff
@@ -13,7 +13,7 @@
 <div class="wp-block-columns alignwide"><!-- wp:column {"verticalAlignment":"stretch","width":"60%","style":{"spacing":{"padding":{"right":"0"}}}} -->
 <div class="wp-block-column is-vertically-aligned-stretch" style="padding-right:0;flex-basis:60%"><!-- wp:group {"style":{"dimensions":{"minHeight":"100%"}},"layout":{"type":"flex","orientation":"vertical","verticalAlignment":"space-between","justifyContent":"stretch"}} -->
 <div class="wp-block-group" style="min-height:100%"><!-- wp:image {"aspectRatio":"9/16","scale":"cover","sizeSlug":"large","linkDestination":"none","className":"is-style-rounded"} -->
-<figure class="wp-block-image size-large is-style-rounded"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/project-1.webp" alt="" style="aspect-ratio:9/16;object-fit:cover"/></figure>
+<figure class="wp-block-image size-large is-style-rounded"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/project-1.webp" alt="<?php echo esc_attr__( 'Project Image', 'twentytwentyfour' ); ?>" style="aspect-ratio:9/16;object-fit:cover"/></figure>
```

Comment:
Please add a proper description of what is visible in the image. A text like this is not helpful for screen readers.